internal/db/gorm.go

```diff
@@ -0,0 +1,66 @@
+package db
+
+import (
+	"database/sql"
+	"errors"
+
+	"github.com/cyverse/QMS/internal/model"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+func InitGORMConnection(db *sql.DB) (*gorm.DB, error) {
+	gormDB, err := gorm.Open(postgres.New(postgres.Config{
+		Conn: db,
+	}), &gorm.Config{
+		DisableForeignKeyConstraintWhenMigrating: true,
```

Comment:
No foreign keys are being added to the database at all. That makes me wary.

Reply:
We I am wondering why it is not being added.
I will try to figure it out.